docs/DeveloperGuide.adoc

```diff
@@ -742,6 +802,33 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 (For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
 
 [discrete]
+=== Use case: Add person
+
+*MSS*
+
+1.  User requests to add a new person
+2.  AddressBook adds the person
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 2a. The person is already in the address book
```

Comment:
Note that for use cases, the numbering is used to describe scenarios that can happen after a step, preventing the user /system from going to the next step. Here it seems to me that because `The person is already in the address book` so the system cannot go to the step which is `AddressBook adds the person`. In that case, `2a` should be `1a`